docs/source/install/upgrades.rst

```diff
@@ -82,6 +88,34 @@ v2.2
   executions_v2, either use psql or install an older version of the python-mistralclient in a
   separate python virtual environment.
 
+.. note::
+
+    Please be sure to follow the general steps listed above to do the database upgrade.
+
+*  If you're seeing an error ``event_triggers_v2 already exists`` when running
+   ``mistral-db-manage upgrade head``, this means the mistral services started before the
```

Comment:
At a high-level this behavior is horrible, so much error-prone and unfriendly in terms of package upgrade story :/